dm/config/subtask.go

```diff
@@ -295,13 +295,13 @@ func (c *SubTaskConfig) DecryptPassword() (*SubTaskConfig, error) {
 	if len(clone.To.Password) > 0 {
 		pswdTo, err = utils.Decrypt(clone.To.Password)
 		if err != nil {
-			return nil, errors.Annotatef(err, "downstream DB password %s", clone.To.Password)
+			return nil, terror.WithScope(terror.ErrConfigDecryptDBPassword.Delegate(err, clone.To.Password), terror.ScopeDownstream)
```

Comment:
decrypting but not executing, _downstream_ or _internal_, which is better?

also for line L304.